specification/carbon/Carbon.Management/client.tsp

```diff
@@ -5,6 +5,22 @@ using Azure.ClientGenerator.Core;
 using Azure.ResourceManager.CommonTypes;
 using Microsoft.Carbon;
 
+@@clientName(Microsoft.Carbon, "CarbonOptimizationMgmt", "python");
+@@clientName(Microsoft.Carbon, "CarbonOptimizationManagementClient", "java");
+
+// CSharp
+@@clientName(CarbonService.queryCarbonEmissionReports,
+  "QueryCarbonEmissionReports",
```

Comment:
why change the first letter to capital case? shouldn't this be handled by the emitter already?

Reply:
Only uncase the first letter works

Reply:
this is strange though, we have so many camel case properties, the .NET SDK properties should be pascal case by default.

Reply:
This renaming for operation is to get rid of the interface name appending.